docs/source/api/v1/riak_ping.rst

```diff
@@ -0,0 +1,65 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+.. _to-api-v1-riak-ping:
+
+*************
+``riak/ping``
+*************
+.. deprecated:: ATCv4
+	Use the ``GET`` method of :ref:`to-api-v1-vault-ping` instead.
```

Comment:
this ref needs to be updated to the new ref tag for vault-ping

Reply:
my bad. done